cluster/saltbase/salt/fluentd-es/fluentd-es.yaml

```diff
@@ -5,20 +5,20 @@ metadata:
 spec:
   containers:
   - name: fluentd-elasticsearch
-    image: gcr.io/google_containers/fluentd-elasticsearch:1.5
-    env:
-    - name: "FLUENTD_ARGS"
-      value: "-qq"
+    image: gcr.io/google_containers/fluentd-elasticsearch:1.6
+    args:
+    - -qq
     volumeMounts:
     - name: varlog
-      mountPath: /varlog
-    - name: containers
+      mountPath: /var/log
```

Comment:
So should this not be changed to `/varlog` ?
You could rename `varlog` to something less confusing.